test/k8s_logging_tests/test_config_logging.py

```diff
@@ -19,7 +19,9 @@ def test_splunk_index(setup, test_input, expected):
     '''
     logging.getLogger().info("testing test_splunk_index input={0} \
                  expected={1} event(s)".format(test_input, expected))
-    search_query = "index=circleci_events OR index=kube-system"
+    default_index = "circleci_events"
+    index_logging = os.environ["CI_INDEX_EVENTS"] if os.environ["CI_INDEX_EVENTS"] else default_index
```

Comment:
Can we define "index_logging", "index_objects", "index_metrics" in the beginning once for all?